spring-cloud-loadbalancer/src/test/java/org/springframework/cloud/loadbalancer/core/TestSelectedServiceInstanceSupplier.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2012-2022 the original author or authors.
```

Comment:
Please change to `* Copyright 2012-2023 the original author or authors.`

Reply:
done